commercetools-models/src/main/java/io/sphere/sdk/productdiscounts/queries/MatchingProductDiscountGet.java

```diff
@@ -0,0 +1,39 @@
+package io.sphere.sdk.productdiscounts.queries;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.productdiscounts.ProductDiscount;
+import io.sphere.sdk.products.Price;
+
+/**
+ * This endpoint can be used to simulate which product discounts would be applied if a specified product variant had a specified price.<br>
+ * Given product and variant ids and a price, this endpoint will return the product discount that would have been applied to that price.
+ */
+public interface MatchingProductDiscountGet extends SphereRequest<ProductDiscount> {
```

Comment:
Here we should extend our `MetaModelGetDsl` interface, see `ProductDiscountByIdGet` for an example. This will then give us support for expansion paths.

Reply:
Just to keep track of the discussion, here we agreed since this request uses post and its an exception to keep it with a minimal implementation as is now.